onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -5126,6 +5126,28 @@ def aten_scatter_add(
     return op.ScatterElements(self, index, src, axis=dim, reduction="add")
 
 
+@torch_op("aten::scatter_reduce", trace_only=True)
+def aten_scatter_reduce(
+    self: TReal,
+    index: TInt,
+    src: TReal,
+    reduce: str,
+    dim: int,
+    include_self: bool = True,  # pylint: disable=unused-argument
```

Comment:
I would match the order from the signature

Reply:
It is hard to use original signature, because the 'dim' input is int, not INT64. It will be used in the following op.ScatterElements( ..... axis=dim.....). So it must be moved down as attribute.

Reply:
We can mix inputs and attributes when trace only because it’s just a plain Python function. We also must match the order because that is what the exporter will expect. In cases when we do want to script parts of the function we can create a trace only interface that matches the argument order with the ATen, which then calls an internal implementation with order adjusted to fit ONNX

Reply:
got your point.